### PR TITLE
Process deprecatedSince annotation

### PR DESCRIPTION
**Pull request by manuelvicnt**
## Description
In the original Javadoc output, deprecatedSince was added like

```
<div class="api-level">
  <div>
    Added in <a href="https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels">API level 1</a>
    <br>
    Deprecated in <a href="https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels">API level 11</a>
  </div>
</div>
```

I added the `@deprecatedSince` processing and the output is as above

## Screenshot of both annotations present
apiSince and deprecatedSince

## Screenshot of only apiSince present
apiSince

## Screenshot of only deprecatedSince present
deprecatedSince

### CODE REVIEW

File: core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt [hunk 1]
```diff
@@ -767,6 +770,30 @@ open class JavaLayoutHtmlFormatOutputBuilder(
         return false
     }
 
+    protected open fun FlowContent.apiAndDeprecatedVersions(node: DocumentationNode) {
+        val apiLevelExists = node.apiLevel.name.isNotEmpty()
+        val deprecatedLevelExists = node.deprecatedLevel.name.isNotEmpty()
+        if (apiLevelExists || deprecatedLevelExists) {
+            div(classes = "api-level") {
+                if (apiLevelExists) {
+                    +"added in "
```
Comment (tiembo) [inline core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt]: added => Added

File: core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt [hunk 2]
```diff
@@ -767,6 +770,30 @@ open class JavaLayoutHtmlFormatOutputBuilder(
...
+                    a(href = "https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels") {
+                        +"API level ${node.apiLevel.name}"
+                    }
+                    if (deprecatedLevelExists) {
+                        br
+                    }
+                }
+                if (deprecatedLevelExists) {
+                    +"deprecated in "
```
Comment (tiembo) [inline core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt]: deprecated => Deprecated

**Review by tiembo — CHANGES_REQUESTED** [review]
LGTM with minor capitalization updates to match Java version of docs

File: core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt (hunk 1 above)
Comment (manuelvicnt) [reply core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt]: Thanks! I'll change it :) I thought it was an intentional change to the Javadoc version

File: core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt (hunk 2 above)
Comment (manuelvicnt) [reply core/src/main/kotlin/Formats/JavaLayoutHtml/JavaLayoutHtmlFormatOutputBuilder.kt]: Thanks! I'll change it :) I thought it was an intentional change to the Javadoc version

**Review by tiembo — APPROVED** [review]

**Pull request merged**